Formula/docker-ls.rb

```diff
@@ -0,0 +1,36 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries."
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :using => :git,
+    :tag => "v0.3.1",
+    :revision => "d80310976c9707e261e57ebfa9acf4e0b1781460"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/mayflower/docker-ls"
+    [
+      buildpath/"cli",
+      buildpath/"generators",
+      buildpath/"lib",
+      buildpath/"vendor",
+      buildpath/".git",
+    ].each do |dir|
+      cp_r dir, buildpath/"src/github.com/mayflower/docker-ls"
+    end
+
+    system "go", "generate", "github.com/mayflower/docker-ls/lib"
+    system "go", "install", "github.com/mayflower/docker-ls/cli/..."
+
+    bin.install buildpath/"bin/docker-ls"
+    bin.install buildpath/"bin/docker-rm"
+  end
+
+  test do
+    system("#{bin}/docker-ls", "version")
```

Comment:
The parens are redundant, in both cases here.

Reply:
Thanks, will change.